### PR TITLE
connect api activity-log

### PR DESCRIPTION
**Pull request by eleanoryy**

### CODE REVIEW

File: src/Views/Dataset/DatasetActivity/DatasetActivity.jsx [hunk 1]
```diff
@@ -62,11 +62,13 @@ const DatasetActivity = (props) => {
                 margin: '0px',
               }}
             >
-              {moment(item.activityTime).format('YYYY-MM-DD HH:MM:SS')}
+              {moment(item.activityTime).local().format('YYYY-MM-DD HH:MM:SS')}
```
Comment (jianglai3303) [inline src/Views/Dataset/DatasetActivity/DatasetActivity.jsx:65]: Please use global timeConvert() function for date display @eleanoryy

File: src/Views/Dataset/DatasetActivity/DatasetActivityLogsDisplay.jsx [hunk 2]
```diff
@@ -435,66 +436,79 @@ const fileInfoDisplay = (caseType, action, details) => {
...
+            download a file/folder:{' '}
+            <span style={{ fontWeight: 600 }}>{details.itemName}</span>
           </p>
         </div>
       );
     } else if (
       action === 'UPDATE' &&
-      details.hasOwnProperty('from') &&
-      details.hasOwnProperty('to')
+      details.changes[0].hasOwnProperty('oldValue') &&
```
Comment (jianglai3303) [inline src/Views/Dataset/DatasetActivity/DatasetActivityLogsDisplay.jsx:483]: add more test for the empty field. e.g. detail.changes && detail.changes.length && details.changes[0].oldValue @eleanoryy

**Review by jianglai3303 — COMMENTED** [review]
@eleanoryy please refactor the related code and submit merge request again
